dojo/settings/settings.dist.py

```diff
@@ -135,7 +135,8 @@
     DD_SLA_NOTIFY_POST_BREACH=(int, 7),
 
     # maximum number of result in search as search can be an expensive operation
-    DD_SEARCH_MAX_RESULTS=(int, 100)
+    DD_SEARCH_MAX_RESULTS=(int, 100),
+    DD_MAX_AUTOCOMPLETE_WORDS=(int, 20000)
```

Comment:
That still seems like a lot? 5000 would still look reasonable?

Reply:
Difference in performance is almost zero between 5kand 20k. 20k seemed to me like a nice seeing that will cover 90% of installs (except yours 🤭)